cmd/fluxd/main.go

```diff
@@ -93,6 +93,7 @@ func main() {
 		gitSkipMessage = fs.String("git-ci-skip-message", "", "additional text for commit messages, useful for skipping builds in CI. Use this to supply specific text, or set --git-ci-skip")
 
 		gitPollInterval = fs.Duration("git-poll-interval", 5*time.Minute, "period at which to poll git repo for new commits")
+		gitTimeOut      = fs.Duration("git-time-out", 20*time.Second, "duration after which git operations time out")
```

Comment:
> I have reservations over using the same timeout for all git operations -- increasing the timeout for git clone might mean that other operations take much longer to fail.

You are not alone with this, I had/have the same concerns. This can be solved eventually by adding a second flag that defines a budget for all operations together (each operation may take `x` but all together it may not take longer than `y`).

---

As for the spelling - will make adjustments later today.